.github/workflows/molecule.yml

```diff
@@ -33,9 +33,6 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Install test requirements
-        run: pip install -r requirements.txt
-
```

Comment:
So this is actually not necessary? If so, please drop daec0cebd0a066437b3a882f10b949675e3201e5.